### ISSUE TITLE
Does not play a sound

### ISSUE THREAD

**Issue by canelhasmateus** [FirstTimeInForever/intellij-music#6]
installed the plugin but it didn't make a sound. tried change settings to enable/disable, play only in editor or not, toggled mute mode, but nothing works.
Running on pycharm pro 2020.1.1

**Comment by eduhoribe** [comment]
I have the same problem in Intellij Ultimate 2020.2 EAP

When the IDE starts, I got the stack trace below:
```
java.io.FileNotFoundException: https://raw.githubusercontent.com/FirstTimeInForever/intellij-music/master/plugin/assets/All_Star.mid
	at java.base/sun.net.www.protocol.http.HttpURLConnection.getInputStream0(HttpURLConnection.java:1915)
	at java.base/sun.net.www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1515)
	at java.base/sun.net.www.protocol.https.HttpsURLConnectionImpl.getInputStream(HttpsURLConnectionImpl.java:250)
	at java.base/java.net.URL.openStream(URL.java:1140)
	at intellij.music.core.UserDirectoryLoader.downloadFile(UserDirectoryLoader.kt:48)
	at intellij.music.core.UserDirectoryLoader.access$downloadFile(UserDirectoryLoader.kt:13)
	at intellij.music.core.UserDirectoryLoader$downloadFontAndMidiFiles$1.run(UserDirectoryLoader.kt:63)
	at com.intellij.openapi.progress.impl.CoreProgressManager$TaskRunnable.run(CoreProgressManager.java:935)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcessWithProgressAsync$5(CoreProgressManager.java:442)
	at com.intellij.openapi.progress.impl.ProgressRunner.lambda$submit$3(ProgressRunner.java:235)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcess$2(CoreProgressManager.java:170)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:629)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:581)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:60)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:157)
	at com.intellij.openapi.progress.impl.ProgressRunner.lambda$submit$4(ProgressRunner.java:235)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1700)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:668)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:665)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1.run(Executors.java:665)
	at java.base/java.lang.Thread.run(Thread.java:834)
```

**Comment by dima74** [comment]
@Demyxter @eduhoribe thanks for reporting and especially for stacktrace! We just released version 1.0.2, where the issue should be fixed
